docs/technical_development_guide/source/changelog/changelog_dev.rst

```diff
@@ -6,6 +6,35 @@ Development Changelog
     :version: v3.5.0
     :released: pending
 
+    .. change::
+        :tags: docs, technical reference guide
+        :pullreq: 1347
+
+        **Date**: 2025-03-12
+
+        Title:
+        Update Technical Reference Guide from 3.3.0 to develop
+
+        Description:
+        Update the Technical Reference Guide based on the PRs from the split-off point of ResStock Release 3.3.0 to current develop.
+
+        Assignees: Anthony Fontanini
+    .. change::
+        :tags: workflow
+        :pullreq: 1347
```

Comment:
I think you want 1353 here; 1347 is already accounted for.

Reply:
Addressed [a57e44d](https://github.com/NREL/resstock/pull/1361/commits/a57e44d9b1f74c62c5c938f9825195e362809d22)